website/content/en/docs/getting-started/getting-started-with-karpenter/scripts/step01-config.sh

```diff
@@ -3,3 +3,5 @@ export CLUSTER_NAME="${USER}-karpenter-demo"
 export AWS_DEFAULT_REGION="us-west-2"
 export AWS_ACCOUNT_ID="$(aws sts get-caller-identity --query Account --output text)"
 export TEMPOUT="$(mktemp)"
+export ARM_AMI_ID="$(aws ssm get-parameter --name /aws/service/eks/optimized-ami/${K8S_VERSION}/amazon-linux-2-arm64/recommended/image_id --query Parameter.Value --output text)"
```

Comment:
Should we also include the GPU AMI?

Reply:
I decided not to, since currently there's no requirements field in the ami from the GPU that would let us know which of these AMIs to choose. WDYT?

Reply:
Ah I see what you mean, maybe it could be included in a similar way to name based selection, commented by default?